apps/web/pages/auth/forgot-password/[id].tsx

```diff
@@ -24,6 +24,8 @@ export default function Page({ requestId, isRequestExpired, csrfToken }: Props)
   const formMethods = useForm<{ new_password: string }>();
   const success = formMethods.formState.isSubmitSuccessful;
   const loading = formMethods.formState.isSubmitting;
+  const [isEmpty, setIsEmpty] = useState<boolean>(true);
```

Comment:
```suggestion
```

Reply:
@Udit-takkar Thank you for reviewing, made suggested changes